pi/imu.py

```diff
@@ -1,59 +1,89 @@
+# SPDX-FileCopyrightText: 2020 Bryan Siepert, written for Adafruit Industries
+#
+# SPDX-License-Identifier: Unlicense
 import time
 import board
 import busio
-import adafruit_bno08x
```

Comment:
Did we push an early version of this last year and that's why this is here. Not block, just curious.

Reply:
i honestly don't remember; this client has been through a few different iterations at this point.

Reply:
thinking on it more, i think this just got replaced with more specific imports.